app/vep/utils/web_metadata.py

```diff
@@ -0,0 +1,26 @@
+from loguru import logger
+import requests
+from typing import List
+from core.config import ENSEMBL_WEB_METADATA_API
+from vep.models.client_model import GenomeAnnotationProvider
+
+async def get_genome_metadata(genome_id: str) -> GenomeAnnotationProvider:
+
+    try:
+      session = requests.Session()
+      with session.get(
+          url=f"https://staging-2020.ensembl.org/api/metadata/genome/{genome_id}/details"
+      ) as response:
+          response.raise_for_status()
+          metadata = response.json()
+    except requests.exceptions.HTTPError as HTTPError:
+      logger.error(f"HTTPError: {HTTPError}")
+      return None
+    except Exception as e:
+      logger.exception(e)
+      return None
+
+    provider = metadata["annotation_provider"]
+    version = metadata["annotation_version"]
```

Comment:
We may not be ready to use the annotation_version yet (Daniel promised to update it with proper data; but right now it may contain the "internal version" not intended for the public). The fallback, I believe, is to use the last updated date.